src/node/mod.rs

```diff
@@ -36,3 +41,32 @@ pub fn init(ct: CancellationToken) -> JoinHandle<()> {
 		info!(target: LOG, "Stopped node agent");
 	})
 }
+
+// Start live query on change feeds notification processing
+pub fn live_query_change_feed(ct: CancellationToken) -> JoinHandle<()> {
+	tokio::spawn(async move {
+		if !FFLAGS.change_feed_live_queries.enabled() {
+			return;
+		}
+		// Spawn the live query change feed consumer, which is used for catching up on relevant change feeds
+		tokio::spawn(async move {
```

Comment:
Why do we need to spawn again here? Why not just continue in the previous thread?

Reply:
Unless you meant future? I think it is easier if it is a "thread" because then we can share impl with wasm. Down the line - atm it's not perfectly shared, but dont have much time to sort that out.

Reply:
I'm not seeing the point of initial thread. All it does is

```rust
if !FFLAGS.change_feed_live_queries.enabled() {
    return;
}
```

That's not even a future we need to await on. Why not just move that out of the thread if you want to make sure it's run first.